src/views/ResetPass.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Header from '../components/global/Header';
+import AuthHolder from '../components/global/AuthHolder';
+
+const ResetPass = (props) => {
+  const { token } = props.match.params;
+  return (
+  <div className="authentication">
+    <Header />
+    <AuthHolder token={token} />
+  </div>
+  );
+};
+
+export default ResetPass;
```

Comment:
Remove this file because there is a component with the same code in /views/authentication.js